prac_06/guitar_test.py

```diff
@@ -0,0 +1,18 @@
+
+from guitar import Guitar
+
+
+def test_guitar():
```

Comment:
It is probably unnecessary to have a function in this testing script.
Consider renaming this function to main() or adding a main() function to run this